<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QMainWindow" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="log_login">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string extracomment="Логин"/>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string>Логин</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="log_reg">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_entry">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="log_password">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string extracomment="Пароль"/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Пароль</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
